lib/str.js

```diff
@@ -24,16 +24,19 @@ const Str = {
     },
 
     /**
-     * Converts a USD string into th number of cents it represents.
+     * Converts a currency string into the number of cents it represents.
      *
-     * @param {String}  amountStr     A string representing a USD value.
+     * @param {String}  amountStr     A string representing a currency symbol and value, like $4.02 or Bs.S97.9
      * @param {Boolean} allowFraction Flag indicating if fractions of cents should be
-     *                               allowed in the output.
+     *                                allowed in the output.
      *
      * @return {Number} The cent value of the @p amountStr.
      */
     fromUSDToNumber(amountStr, allowFraction) {
-        let amount = String(amountStr).replace(/[^\d.\-()]+/g, '');
+        amountStr = String(amountStr);
+        // Currently, no currency symbol has a number in it, so we find the first number
+        let withoutCurrency = amountStr.substring(amountStr.search(/\d/));
```

Comment:
Will amounts always have a leading zero? like would we ever pass `$.25` or something like that? If so this might return the wrong value. (I'm not sure on the answer to this)